samples/TagHelperSample.Web/Startup.cs

```diff
@@ -4,23 +4,32 @@
 using Microsoft.AspNet.Builder;
 using Microsoft.AspNet.Mvc;
 using Microsoft.Framework.DependencyInjection;
+using Microsoft.Framework.Logging;
+using Microsoft.Framework.Logging.Console;
 using TagHelperSample.Web.Services;
 
 namespace TagHelperSample.Web
 {
     public class Startup
     {
-        public void Configure(IApplicationBuilder app)
+        public void Configure(IApplicationBuilder app, ILoggerFactory loggerFactory)
         {
-            app.UseServices(services =>
-            {
-                services.AddMvc();
-                
-                // Setup services with a test AssemblyProvider so that only the sample's assemblies are loaded. This
-                // prevents loading controllers from other assemblies when the sample is used in functional tests.
-                services.AddTransient<IAssemblyProvider, TestAssemblyProvider<Startup>>();
-                services.AddSingleton<MoviesService>();
-            });
+            app.UseServices(services => 
+            { 
+                services.AddMvc(); 
+                 
+                // Setup services with a test AssemblyProvider so that only the sample's assemblies are loaded. This 
+                // prevents loading controllers from other assemblies when the sample is used in functional tests. 
+                services.AddTransient<IAssemblyProvider, TestAssemblyProvider<Startup>>(); 
+                services.AddSingleton<MoviesService>(); 
+            }); 
+
+            loggerFactory.AddConsole((name, logLevel) => name.IndexOf("TagHelper") >= 0 || name.IndexOf("WebListener") >= 0 && logLevel >= LogLevel.Information);
+            
+            app.UseErrorPage();
```

Comment:
Because if you use self-host, which is the easiest way to see the trace messages, and you have an error in the site, you just get a blank page. We don't always have the pretty error page that Helios gives us.